LICENSE.txt

```diff
@@ -2240,3 +2241,46 @@ be copied, modified, or distributed except according to those terms.
 
 Distributed on an "AS IS" BASIS, WITHOUT WARRANTY OF ANY KIND, either
 express or implied.  See your chosen license for details.
+
+--------------------------------------------------------------------------------
+
+The file cpp/src/arrow/util/bit_stream_utils.h contains code from
```

Comment:
there is already a reference to protocol buggers above, could you update that instead?